### PR TITLE
A few editorial changes

### PR DESCRIPTION
**Pull request by littledan**

### CODE REVIEW

**Comment by littledan** [comment]
I think it wasn't the best idea to switch the pattern to being "pre-parsed" the way this PR does; better to leave it in pattern form to match DateTimeFormat and NumberFormat. But, at the same, time, it could match the more closely, rather than referring to RegExps (in a way which is not done anywhere else in ECMA262)--we could use ECMAScript grammars here, but maybe that could be done separately. I'll upload a new version which is a revert and cleanup of the old logic.

(I noticed some other places with room for improvement--for example, it seems like \[\[LocaleData]] is supposed to include a pluralRules property, but this isn't set up or used. I'll also fix this in a follow-on.)

File: spec/relativetimeformat.html [hunk 1]
```diff
@@ -73,68 +72,57 @@
...
+        1. Assert: _length_ is a non-negative integer.
+        1. Repeat while _beginIndex_ is an integer index into _pattern_:
+          1. Set _endIndex_ to Call(%StringProto_indexOf%, _pattern_, "}", _beginIndex_)
+          1. Assert: _endIndex_ is not -1, otherwise the pattern would be malformed.
+          1. If _beginIndex_ is greater than _nextIndex_, then:
+            1. Let _literal_ be a substring of _pattern_ from position _nextIndex_, inclusive, to position _beginIndex_, exclusive.
+            1. Add new part record { [[Type]]: `"literal"`, [[Value]]: _literal_ } as a new element of the list _result_.
+          1. Let _p_ be the substring of _pattern_ from position _beginIndex_, exclusive, to position _endIndex_, exclusive.
+          1. Assert: _p_ is `"0"`.
+          1. Add new part Record { [[type]]: _type_, [[value]]: _value_ } as a new element on the List _result_.
```
Comment (zbraniecki) [inline spec/relativetimeformat.html]: Capital `Type` and `Value`

Comment (zbraniecki) [inline spec/relativetimeformat.html]: where do you get `_type_` and `_value_` from?

**Comment by zbraniecki** [comment]
Overall it looks great! Thanks @littledan for picking up my half-baked `DeconstructPattern` function and turning it into a real spec algo :)

File: spec/relativetimeformat.html (hunk 1 above)
Comment (littledan) [inline spec/relativetimeformat.html]: Capitalized it. _value_ is a parameter, and _type_ was supposed to be a parameter that I was halfway through renaming to _unit_. Now it's consistently renamed.

**Comment by littledan** [comment]
@zbraniecki Note that this wording loses the nice generality that you created before. I have some ideas for restoring that, but I need to think about it some more.

**Pull request merged**